compass/landice/tests/calving_dt_convergence/albany_input.yaml

```diff
@@ -0,0 +1,265 @@
+%YAML 1.1
```

Comment:
I don't understand where this file is used. I see that the Humboldt, Thwaites, and MISMIP+ cases all use albany_input.yaml files that are in their respective test group directories.